src/Core/SharedKernels/Enumerator/WB.Enumerator.Native/WebInterview/Controllers/InterviewDataController.cs

```diff
@@ -99,7 +99,47 @@ public virtual InterviewInfo GetInterviewDetails(Guid interviewId)
                 CanAddComments = statefulInterview.Status != InterviewStatus.ApprovedByHeadquarters && !this.IsCurrentUserObserving(),
                 ReceivedByInterviewer = statefulInterview.ReceivedByInterviewer,
                 IsCurrentUserObserving = this.IsCurrentUserObserving(),
-                DoesBrokenPackageExist = false
+                DoesBrokenPackageExist = false,
+                IsExistsCriticality =  questionnaire.IsExistsCriticality(),

```

Comment:
HasCriticalRules?
CriticallyEvaluatable?

Reply:
renamed to doesSupportCriticality